stellar-soroban-rpc/debian/stellar-soroban-rpc.default

```diff
@@ -0,0 +1,17 @@
+## /etc/default/stellar-soroban-rpc
+
+# define the port on local machine that soroban rpc server will bind to.
+ENDPOINT=localhost:8003
+# for now rpc requires access to horizon, this will change.
+HORIZON_URL=http://localhost:8001
```

Comment:
Should we use default port 8000 that horizon uses?

Reply:
yes, good catch, changed it - https://github.com/stellar/packages/pull/183/commits/bed3eef1b520e521cd4886c085e08dc553abdce6